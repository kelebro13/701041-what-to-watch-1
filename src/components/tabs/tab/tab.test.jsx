import renderer from "react-test-renderer";
import Tab from "./tab";

it(`renders property for selected`, () => {
  const wrapper = renderer
    .create(<Tab index={1} title={`Test`} isSelected={true}/>)
    .toJSON();

  expect(wrapper).toMatchSnapshot();
});

it(`renders property for not selected`, () => {
  const wrapper = renderer
    .create(<Tab index={1} title={`Test`} isSelected={false}/>)
    .toJSON();

  expect(wrapper).toMatchSnapshot();
});
