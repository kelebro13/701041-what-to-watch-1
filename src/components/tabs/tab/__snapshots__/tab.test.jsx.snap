// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders property for not selected 1`] = `
<li
  className="movie-nav__item "
>
  <a
    className="movie-nav__link"
    href="#"
    onClick={[Function]}
  >
    Test
  </a>
</li>
`;

exports[`renders property for selected 1`] = `
<li
  className="movie-nav__item movie-nav__item--active"
>
  <a
    className="movie-nav__link"
    href="#"
    onClick={[Function]}
  >
    Test
  </a>
</li>
`;
